### ISSUE TITLE
MySQL parse error on timestamp column with "ON UPDATE CURRENT TIMESTAMP"

### ISSUE THREAD

**Issue by TooBug** [taozhi8833998/node-sql-parser#968]
**Describe the bug**

Parse error on timestamp column with "ON UPDATE CURRENT TIMESTAMP"

```
Expected "#", ")", ",", "--", "/*", "AUTO_INCREMENT", "CHARACTER", "CHECK", "COLLATE", "COLUMN_FORMAT", "COMMENT", "CONSTRAINT", "DEFAULT", "KEY", "NOT NULL", "NULL", "PRIMARY", "REFERENCES", "STORAGE", "UNIQUE", or [ \t\n\r] but "O" found.
```

**Database Engine**

MySQL

**To Reproduce**

```sql
CREATE TABLE `t1` (
  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
  `name` varchar(64) NOT NULL DEFAULT 'ttt',
  `zf` int(10) unsigned zerofill DEFAULT NULL,
  `created_at` timestamp NULL DEFAULT NULL,
  `updated_at` timestamp NULL DEFAULT NULL ON UPDATE CURRENT_TIMESTAMP,
  PRIMARY KEY (`id`)
) ENGINE=InnoDB AUTO_INCREMENT=5 DEFAULT CHARSET=utf8mb4
```

node-sql-parser version: 4.1.1
node version: 16.3.0

**Comment by taozhi8833998** [comment]
@TooBug fixed in the newer version, upgrade to the `4.2.0`.

**Comment by TooBug** [comment]
@taozhi8833998 Great work, thanks!
